reports/imports/import_2010.py

```diff
@@ -0,0 +1,299 @@
+from .base import BaseImport, v
+
+from reports.utils.canon import canon
+
+class Import2010(BaseImport):
+    """
+        Holds the methods that handles importing 2010 data files
+    """
+    def get_work_sheet(self, wb, old_sheet, new_sheet):
+        ws = None
+        for name in wb.sheetnames:
+            if name == old_sheet or name.startswith(old_sheet + ' '):
+                ws = wb[name]
+        return ws
```

Comment:
Since we have a class now, why not `ws` as part of `self` so that we don't have to pass it around in every method?